Headlines/AppDelegate.swift

```diff
@@ -58,6 +60,61 @@ class AppDelegate: UIResponder,
         }
     }
     
+    func retryFetchCategoriesAndNewsAlert(with error: NSError) {
+        var window: UIWindow? = UIWindow(frame: UIScreen.main.bounds)
+        window?.rootViewController = UIViewController()
+        window?.windowLevel = UIWindowLevelAlert + 1
+        
+        let alert = UIAlertController(
+            title: "Whoops",
+            message: error.localizedDescription,
+            preferredStyle: .alert
+        )
+        
+        let retry = UIAlertAction(
+            title: "Volver a intentar",
+            style: .default) { [unowned self] (action) in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/a2d27b0116f8954023234cf50bd24bedd1aa58d7/Headlines/AppDelegate.swift#L76"></span>Unused parameter “action” in a closure should be replaced with _.</td>
    </tr>
  </tbody>
</table>